blog/v4_announcement.ipynb

```diff
@@ -8,47 +8,30 @@
     "(v4_announcement)=\n",
     "\n",
     "# PyMC 4.0 Release Announcement\n",
-    "\n",
-    ":::{post} Jun 10, 2022\n",
+    ":::{post} June 6, 2022\n",
     ":tags: release, aesara, jax\n",
     ":category: news\n",
     ":author: Thomas Wiecki\n",
-    ":::\n",
     "\n",
-    "We, the PyMC core development team, are incredibly excited to announce the release of a major rewrite of PyMC3 (now called just PyMC): `4.0`. This marks the first major new version in over 10 years. Internally, we have already been using PyMC 4.0 almost exclusively for many months and found it to be very stable and better in every aspect. Every user should upgrade, as there are many exciting new updates that we will talk about in this and upcoming blog posts. \n",
+    "We, the PyMC core development team, are incredibly excited to announce the release of a major rewrite of PyMC3 (now called just PyMC): `4.0`. Internally, we have already been using PyMC 4.0 almost exclusively for many months and found it to be very stable and better in every aspect. Every user should upgrade, as there are many exciting new updates that we will talk about in this and upcoming blog posts.\n",
     "\n",
     ":::{figure-md} versions_diagram\n",
-    "\n",
     "![Diagram of the PyMC version history](pymc_versions.png)\n",
-    "\n",
     "Graphic by [Ravin Kumar](https://twitter.com/canyon289)\n",
-    ":::\n",
```

Comment:
It looks like this is deleting the emtpy lines in the figure directive which are needed for the image to render.

It appears now in the generated blog post: https://www.pymc.io/blog/pymc3_vs_pymc_v4.html#v4-announcement but doesn't show in the preview for this PR: https://pymcio--28.org.readthedocs.build/blog/v4_announcement.html#v4_announcement. I am not sure if the image should be there or not but either way a change is needed. if it should the spaces are missing, if it should not then everything should be gone, not only the empty lines

Reply:
Should it be 
```
:::{figure-md} versions_diagram
![Diagram of the PyMC version history](pymc_versions.png)
Graphic by [Ravin Kumar](https://twitter.com/canyon289)
:::
```

Or
```
:::{figure-md} versions_diagram

![Diagram of the PyMC version history](pymc_versions.png)

Graphic by [Ravin Kumar](https://twitter.com/canyon289)
:::
```
?

Reply:
Tried to fix it.

Reply:
it might work without the space between the upper colon fence and the image, but it definitely needs the space between image and caption